R/collect-rd.R

```diff
@@ -0,0 +1,17 @@
+#' @title Create, modify, and delete columns (duckplyr)
+#'
+#' @description  This is a method for the [`dplyr::collect()`] generic.
+#' `collect()` converts the input to a tibble, materializing any lazy operations.
```

Comment:
The phrase "materializing any lazy operations" is not necessarily clear. 

I see it's also present in the error messages. Now, maybe it's a very common term when working with databases. However, isn't duckplyr meant to be used by people who don't necessarily know a lot about databases?

The dplyr manual page has a nice phrasing: "retrieves data into a local tibble". For collapse() it mentions the idea of "forcing computation".